### ISSUE TITLE
Question about the random seed for reproducibility

### ISSUE THREAD

**Issue by Hosein47** [POSTECH-CVLab/PyTorch-StudioGAN#132]
Hello @MINGUKKANG,

I was wondering if you could elaborate on your random seed choices for your model. I mean how many test runs did you do on your model with different seeds? Moreover, how was the effect of the change? Did it prompt a dramatic divergence of the model? And lastly, what is your experience-based suggestion on having a good reproducibility on random seeds?

Cheers,
Hosein

**Comment by MINGUKKANG** [comment]
I was suffering from a high fever these days, so I couldn't answer the question. Thanks for your patience. 

**About random seed choice**
When I train a GAN model, I just choose a randomly sampled seed.

** Number of train runs**
In my experience, it does not seem very important to select a good random seed for successful GAN training. Instead, I tried to find the appropriate hyper-parameter settings. However, if you use mixed-precision representations, I suggest that you train the model at least three times. The performance difference range from 3% to over 100% based on the mean FID value.

** Number of test runs**
Fortunately, performance variations caused by a test time seed are not that large. I usually evaluate trained models only once. 

Thank you.

Best,

Minguk

**Comment by Hosein47** [comment]
Hi @MINGUKKANG, I am also sorry for the late response. I hope you have recovered from the illness. 
Thank you for your answer. I also followed your suggestion in our case. Our work will be out soon hopefully.

Cheers,
Hosein
